management-api-agent-4.x/src/main/java/com/datastax/mgmtapi/shim/CassandraAPI4x.java

```diff
@@ -67,6 +68,33 @@
 
     private static final Supplier<SeedProvider> seedProvider = Suppliers.memoize(() -> new K8SeedProvider4x());
 
+    public void enableFullQuerylog() 
```

Comment:
nit: Add the `@Override` annotation on these three methods. Just a style thing :wink:

Reply:
Fair. Will do.